data_to_pinecone_llm/infra/airbyte/main.tf

```diff
@@ -0,0 +1,98 @@
+// Airbyte Terraform provider documentation: https://registry.terraform.io/providers/airbytehq/airbyte/latest/docs
+
+// Sources
+resource "airbyte_source_bigquery" "bigquery" {
+    configuration = {
+      credentials_json = "...my_credentials_json..."
+      dataset_id       = "...my_dataset_id..."
+      project_id       = "...my_project_id..."
```

Comment:
These can all be set as terraform variables in `variables.tf`, which will cause them to be prompted at runtime if not provided elsewhere.

Terraform can also add a friendly prompt corresponding to a `description` of the variable, and support secrets by adding a `sensitive = true` declaration inside the variable declaration.

E.g.:

```hcl
variable "db_username" {
  description = "Database administrator username"
  type        = string
  sensitive   = true
}
```

From: https://developer.hashicorp.com/terraform/tutorials/configuration-language/sensitive-variables#refactor-database-credentials

Reply:
Nice, thanks! I've moved most configuration items into variables & made keys/tokens/secrets sensitive